index.html

```diff
@@ -56,49 +53,38 @@ <h2>Introduction to EVM Object Format</h2>
 						<!-- First Section -->
 							<section id="first" class="main special">
 								<header class="major">
-									<h2>EOF EIPs</h2>
+									<h2>Why?</h2>
 								</header>
 								<ul class="features">
 									<li>
-										<span class="icon solid major style1 fa-check"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-3541">EIP-3541</a></h3>
-                    <p>Rejects deploying code starting with the byte 0xef. This EIP is already live on Ethereum Mainnet</p>
+                    <h3>Code validation at deploy time</h3>
+                    <p>Instead of having the intepreter validate instructions repeatedly at runtime, EVM bytecode is validated once and during contract creation, making for a faster, cheaper, and more secure execution.</p>
 									</li>
 									<li>
-										<span class="icon major style3 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-3540">EIP-3540</a></h3>
-                    <p><b>EOF - EVM Object Format</b>.</br> Introduces the EOF container format with once-off validation at deploy time.</p>
+                    <h3>Code-data separation</h3>
+                    <p>EOF separates EVM bytecode from data, making it easier for static analysis tools, formal verification, and L2s to process executable code.</p>
 									</li>
 									<li>
-										<span class="icon major style5 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-3670">EIP-3670</a></h3>
-                    <p><b>EOF - Code Validation</b>.</br> Defines how EOF bytecode is validated for correctness at deploy time.</p>
+                    <h3>Remove dynamic JUMPs</h3>
+                    <p>EOF bytecode uses only Static Relative JUMPs, which is very desirable for tools involving static analysis and formal verification, as well as L2 EVMs. </p>
 									</li>
-								</ul>
-                <ul class="features">
 									<li>
-										<span class="icon major style2 fa-file-code"></span>
-                    <h3><a href="https://eips.ethereum.org/EIPS/eip-4200">EIP-4200</a></h3>
-                    <p><b>EOF - Static Relative Jumps</b>.</br> Introduces new instructions: RJUMP, RJUMPI, and RJUMPV with a signed immediate encoding the jump destination.</p>
+                    <h3>Code and gas non-observability</h3>
+                    <p>Lack of code and gas observability is a <a href="https://ethereum-magicians.org/t/eof-proposal-ban-code-introspection-of-eof-accounts/12113">long-sought trait of the EVM</a> allowing for more maintainable and upgradeable protocol.</p>
```

Comment:
Can we have a reference section and citation-like annotations for links to articles/mentions?

Reply:
For a website, unlike spec, I'd do hyperlinks to keep things informal and approachable.